pkgs/os-specific/darwin/apple-source-releases/system_cmds/patches/0001-Add-availability-checks-to-vm_purgeable_stat.patch

```diff
@@ -0,0 +1,42 @@
+From 18a676bc44a8bc24374ab35a258a18049b56b6d4 Mon Sep 17 00:00:00 2001
```

Comment:
Nope. macOS 11.**3**. This will have to stay until 12.0 becomes the minimum.

Reply:
Hmm. We’ve announced that we are requiring macOS 11. We didn’t say anything about the minor version. Given that the last version of Big Sur was 11.7, and new minor versions don’t change hardware support or make meaningful breaking changes, I think it’s perfectly reasonable for us to require 11.3 for the deployment target, which will also maintain the invariant that our default SDK version is also our default deployment target, so that things that don’t do availability checks work as expected. So I think this *can* go in 25.05. Does that sound reasonable?